ansible/roles/stack-sunbird/templates/sunbird_learner-service.env

```diff
@@ -103,3 +103,7 @@ sunbird_user_cert_kafka_topic="{{env_name}}.user.cert.raw"
 # Release-2.3.0
 sunbird_reset_pass_msg={{sunbird_reset_pass_msg | d('You have requested to reset password. Click on the link to set a password: {0}')}}
 sunbird_fuzzy_search_threshold={{sunbird_fuzzy_search_threshold | d('0.5')}}
+sunbird_subdomain_keycloak_base_url={{sunbird_subdomain_keycloak_base_url}}
+#Required for cert-service
+DOMAIN_URL={{DOMAIN_URL}}
```

Comment:
@manzarul Please use small case for variables as per the format of exisiting variables.
domain_url={{proxy_server_name}}       # This will give a value like dev.sunbirded.org

If you require https, then use it as below
domain_url={{proto}}://{{proxy_server_name}}